src/wasm-binary.h

```diff
@@ -155,7 +155,8 @@ enum Section {
   Functions = 2,
   DataSegments = 4,
   FunctionTable = 5,
-  End = 6
+  End = 6,
+  WLLSegment = 0x11
```

Comment:
I'm trying to meet you halfway here.

I don't think Binaryen should support WLL, because I know nothing about it except for the name, and it is not something that is mainstream in the wasm design process.

On the other hand, I am trying to compromise with you, because all you want is to skip a certain section. And since I know skipping sections is something we want anyhow, I suggested that we make this a generic "unrecognized section", and it would be skipped.